package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@trycourier/courier",
-  "version": "3.9.0",
+  "version": "3.10.0",
```

Comment:
@suhasdeshpande did we miss a version? changelog says 3.11

Reply:
maybe @lednax?? This should be `3.11.0` I saw @lednax merged something

Reply:
```suggestion
  "version": "3.11.0",
```